console_conf/controllers/identity.py

```diff
@@ -116,10 +117,21 @@ def write_login_details(fp, username, ips):
         )
     else:
         first_ip = ips[0]
+        key_info = None
+        if os.getenv("SNAP_CONFINEMENT", "classic") == "strict":
+            # if we run in confinement, we have no direct accesss to host
+            # keys info use prepared finger prints if exist
+            host_fingerprints_path = "/run/console-conf/host-fingerprints.txt"
+            host_fingerprints = Path(host_fingerprints_path)
+            if host_fingerprints.is_file():
+                fingerprints = open(host_fingerprints_path, "r")
```

Comment:
`open` is perfectly content to open a Pathlib object, so we might as well drop the host_fingerprints_path value.